core/src/html/layout.rs

```diff
@@ -523,11 +522,19 @@ impl<'a, 'gc> LayoutContext<'a, 'gc> {
 
         // TODO: handle multiple fonts for a definition, each covering different sets of glyphs
 
-        // Use the default serif font, when no other matching font was found.
+        // Font name was neither one of the three default fonts nor matched any device font.
+        // As a last resort fallback to the default serif font, but as a slight compatibility
```

Comment:
I had to read this sentence three times before successfully parsing it... 😅
I read the first part as "As a last resort fallback to the default serif font, <do something that would be a fallback for the default serif font?>".
To me, "fallback" is a noun and in this case I would have written "fall back".
But I not English good, so it's probably okay as is.

Reply:
Alternatively, a comma after "resort" would make it clearer, too.

Reply:
Fallback is a noun. But I just reworded the whole comment, hopefully it's clearer now?

Reply:
Yes, it is clear now. Thanks!